doc/tutorials/using_serial_port.rst

```diff
@@ -41,3 +44,64 @@ to connect to your host system.
 
 Note that If you want to use the RS232 DB9 female/female cable, choose
 the **cross-over** type rather than **straight-through** type.
+
+Option 2: Enabling PCIe Serial Port on TGL Intel NUC
+====================================================
+
+If there is no internal serial port header on your NUC; you can enable PCIe serial port to debug the issues effeciently.
+As you know,most of the NUC don't have PCIe interface,but have NVMe M.2 interface;so the additional M.2 to PCIe adaptor is needed to 
+enabnle PCIe serial port on NUC.You can buy the `adaptor 
+<https://item.jd.com/10025455296900.html>`_ first; Then buy the PCIe serial card with `StarTech 2 Port Native PCI Express 
+<https://www.ebay.ca/i/351912927278>`_  or `IO-PCE99100-2S.
+<https://item.jd.com/1126612955.html>`_ 
+Both of them are verified with this `commit.
+<https://github.com/projectacrn/acrn-hypervisor/commit/9e838248c3ce4d6b68e1c5b068d10d566a06db10>`_
+
+.. figure:: images/NVMe-M.2-to-PCIe-adaptor.png
+   :scale: 80
+
+   NVMe M.2 to PCIe adaptor
+
+.. figure:: images/PCIe-serial-StarTech.png
+   :scale: 80
+
+   StarTech 2 Port Native PCI Express
+
+.. figure:: images/PCIe-serial-LeKuo.png
+   :scale: 80
+
+   IO-PCE99100-2S
+
+    
+Check the BDF Information
+*************************
+Connection as following,boot into native and check the bdf information of the adapter using: ``lspci``
+
+.. figure:: images/PCIe-serial-Connection.png
+   :scale: 80
+
+   PCIe Serial Connection on NUC
+
+.. code-block:: bash
+   
+   01:00.0 Serial controller: Asix Electronics Corporation Device 9100
+   01:00.1 Serial controller: Asix Electronics Corporation Device 9100
+
+Convert the BDF to Hex Format
+*****************************
+
+Refer this :ref:`hv-parameters` to change bdf 01:00.1 to Hex format: 0x101;
+Then adding it into grub menu:
+
+.. Note::
```

Comment:
I did not mean to change that into a note, I was questioning the use of `bash` as the `code-block` style.